From: Didier Raboud <odyx@debian.org>
Date: Tue, 9 Aug 2016 18:11:05 +0200
Subject: Fix hyphen-used-as-minus-sign manpage errors

Automatically generated using sed -i -s 's/^-/\\-/g;s/ -/ \\-/g;s/"-/"\\-/g' man/*.man*
Also fix a missing backslash escape

Origin: vendor
Bug-Upstream: https://github.com/apple/cups/issues/4299
Patch-Name: manpage-hyphen-minus.patch
---
 man/backend.7         |  4 ++--
 man/client.conf.5     |  4 ++--
 man/cups-files.conf.5 |  2 +-
 man/cups-lpd.8        |  6 +++---
 man/cups-snmp.conf.5  |  2 +-
 man/cups.1            |  2 +-
 man/cupsctl.8         |  6 +++---
 man/cupsd-logs.5      | 36 ++++++++++++++++++------------------
 man/cupsd.conf.5      |  2 +-
 man/cupsfilter.8      |  2 +-
 man/filter.7          |  2 +-
 man/ippeveprinter.1   |  6 +++---
 man/ippfind.1         |  6 +++---
 man/ipptoolfile.5     |  8 ++++----
 man/lp.1              | 12 ++++++------
 man/lpadmin.8         |  6 +++---
 man/lpoptions.1       |  4 ++--
 man/lpr.1             |  8 ++++----
 man/mime.convs.5      |  6 +++---
 man/mime.types.5      |  4 ++--
 man/ppdc.1            |  2 +-
 21 files changed, 65 insertions(+), 65 deletions(-)

diff --git a/man/backend.7 b/man/backend.7
index 5a70326..3c42631 100644
--- a/man/backend.7
+++ b/man/backend.7
@@ -163,7 +163,7 @@ The scheduler will respond to this by canceling the job.
 .TP 5
 .B CUPS_BACKEND_RETRY
 The print file was not successfully transmitted because of a temporary issue.
-The scheduler will retry the job at a future time - other jobs may print before this one.
+The scheduler will retry the job at a future time \- other jobs may print before this one.
 .TP 5
 .B CUPS_BACKEND_RETRY_CURRENT
 The print file was not successfully transmitted because of a temporary issue.
@@ -198,7 +198,7 @@ or
 programs to send print jobs or
 .BR lpinfo (8)
 to query for available printers using the backend.
-The one exception is the SNMP backend - see
+The one exception is the SNMP backend \- see
 .BR cups-snmp (8)
 for more information.
 .SH NOTES
diff --git a/man/client.conf.5 b/man/client.conf.5
index 4dbf0c4..f0adb9f 100644
--- a/man/client.conf.5
+++ b/man/client.conf.5
@@ -77,7 +77,7 @@ The \fIAllowDH\fR option enables cipher suites using plain Diffie-Hellman key ne
 The \fIAllowRC4\fR option enables the 128-bit RC4 cipher suites, which are required for some older clients.
 The \fIAllowSSL3\fR option enables SSL v3.0, which is required for some older clients that do not support TLS v1.0.
 The \fIDenyCBC\fR option disables all CBC cipher suites.
-The \fIDenyTLS1.0\fR option disables TLS v1.0 support - this sets the minimum protocol version to TLS v1.1.
+The \fIDenyTLS1.0\fR option disables TLS v1.0 support \- this sets the minimum protocol version to TLS v1.1.
 The \fIMinTLS\fR options set the minimum TLS version to support.
 The \fIMaxTLS\fR options set the maximum TLS version to support.
 Not all operating systems support TLS 1.3 at this time.
@@ -130,7 +130,7 @@ Configuration settings can instead be viewed or changed using the
 command:
 .nf
 defaults write /Library/Preferences/org.cups.PrintingPrefs.plist Encryption Required
-defaults write /Library/Preferences/org.cups.PrintingPrefs.plist TrustOnFirstUse -bool NO
+defaults write /Library/Preferences/org.cups.PrintingPrefs.plist TrustOnFirstUse \-bool NO
 
 defaults read /Library/Preferences/org.cups.PrintingPrefs.plist Encryption
 .fi
diff --git a/man/cups-files.conf.5 b/man/cups-files.conf.5
index b451e2f..7249406 100644
--- a/man/cups-files.conf.5
+++ b/man/cups-files.conf.5
@@ -236,7 +236,7 @@ The following directives are deprecated and will be removed from a future versio
 \fBFileDevice No\fR
 Specifies whether the file pseudo-device can be used for new printer queues.
 The URI "file:///dev/null" is always allowed.
-File devices cannot be used with "raw" print queues - a PPD file is required.
+File devices cannot be used with "raw" print queues \- a PPD file is required.
 The specified file is overwritten for every print job.
 Writing to directories is not supported.
 .\"#FontPath
diff --git a/man/cups-lpd.8 b/man/cups-lpd.8
index e8ce325..344d2af 100644
--- a/man/cups-lpd.8
+++ b/man/cups-lpd.8
@@ -15,9 +15,9 @@ cups-lpd \- receive print jobs and report printer status to lpd clients (depreca
 [
 \fB\-h \fIhostname\fR[\fB:\fIport\fR]
 ] [
-.B -n
+.B \-n
 ] [
-.B -o
+.B \-o
 .I option=value
 ]
 .SH DESCRIPTION
@@ -34,7 +34,7 @@ and
 \fB-h \fIhostname\fR[\fB:\fIport\fR]
 Sets the CUPS server (and port) to use.
 .TP 5
-.B -n
+.B \-n
 Disables reverse address lookups; normally
 .B cups-lpd
 will try to discover the hostname of the client via a reverse DNS lookup.
diff --git a/man/cups-snmp.conf.5 b/man/cups-snmp.conf.5
index 4ca20f2..bb77c73 100644
--- a/man/cups-snmp.conf.5
+++ b/man/cups-snmp.conf.5
@@ -16,7 +16,7 @@ The
 file configures how the standard CUPS network backends (http, https, ipp, ipps, lpd, snmp, and socket) access printer information using SNMPv1 and is normally located in the \fI/etc/cups\fR directory.
 Each line in the file can be a configuration directive, a blank line, or a comment. Comment lines start with the # character.
 .LP
-The Community and DebugLevel directives are used by all backends. The remainder apply only to the SNMP backend -
+The Community and DebugLevel directives are used by all backends. The remainder apply only to the SNMP backend \-
 .BR cups-snmp (8).
 .SH DIRECTIVES
 The following directives are understood by the CUPS network backends:
diff --git a/man/cups.1 b/man/cups.1
index 706620d..b0c51a7 100644
--- a/man/cups.1
+++ b/man/cups.1
@@ -38,7 +38,7 @@ The easiest way to start is by using the web interface to configure your printer
 .LP
 When you are asked for a username and password, enter your login username and password or the "root" username and password.
 .LP
-After the printer is added you will be asked to set the default printer options (paper size, output mode, etc.) for the printer. Make any changes as needed and then click/press on the Set Default Options button to save them. Some printers also support auto-configuration - click/press on the Query Printer for Default Options button to update the options automatically.
+After the printer is added you will be asked to set the default printer options (paper size, output mode, etc.) for the printer. Make any changes as needed and then click/press on the Set Default Options button to save them. Some printers also support auto-configuration \- click/press on the Query Printer for Default Options button to update the options automatically.
 .LP
 Once you have added the printer, you can print to it from any application. You can also choose Print Test Page from the maintenance menu to print a simple test page and verify that everything is working properly.
 .LP
diff --git a/man/cupsctl.8 b/man/cupsctl.8
index 791954c..c88eb22 100644
--- a/man/cupsctl.8
+++ b/man/cupsctl.8
@@ -73,19 +73,19 @@ Display the current settings:
 Enable debug logging:
 .nf
 
-    cupsctl --debug-logging
+    cupsctl \--debug-logging
 
 .fi
 Get the current debug logging state:
 .nf
 
-    cupsctl | grep '^_debug_logging' | awk -F= '{print $2}'
+    cupsctl | grep '^_debug_logging' | awk \-F= '{print $2}'
 
 .fi
 Disable printer sharing:
 .nf
 
-    cupsctl --no-share-printers
+    cupsctl \--no-share-printers
 .fi
 .SH KNOWN ISSUES
 You cannot set the Listen or Port directives using \fBcupsctl\fR.
diff --git a/man/cupsd-logs.5 b/man/cupsd-logs.5
index 2070be9..a912c46 100644
--- a/man/cupsd-logs.5
+++ b/man/cupsd-logs.5
@@ -30,25 +30,25 @@ Each line is in an extended version of the so-called "Common Log Format" used by
 For example:
 .nf
 
-    10.0.1.2 - - [01/Dec/2005:21:50:28 +0000] "POST / HTTP/1.1" 200 317
+    10.0.1.2 \- \- [01/Dec/2005:21:50:28 +0000] "POST / HTTP/1.1" 200 317
       CUPS-Get-Printers successful-ok-ignored-or-substituted-attributes
-    localhost - - [01/Dec/2005:21:50:32 +0000] "GET /admin HTTP/1.1"
-      200 0 - -
-    localhost - - [01/Dec/2005:21:50:32 +0000] "POST / HTTP/1.1"
+    localhost \- \- [01/Dec/2005:21:50:32 +0000] "GET /admin HTTP/1.1"
+      200 0 \- \-
+    localhost \- \- [01/Dec/2005:21:50:32 +0000] "POST / HTTP/1.1"
       200 157 CUPS-Get-Printers
       successful-ok-ignored-or-substituted-attributes
-    localhost - - [01/Dec/2005:21:50:32 +0000] "POST / HTTP/1.1"
-      200 1411 CUPS-Get-Devices -
-    localhost - - [01/Dec/2005:21:50:32 +0000] "GET /admin HTTP/1.1"
-      200 6667 - -
+    localhost \- \- [01/Dec/2005:21:50:32 +0000] "POST / HTTP/1.1"
+      200 1411 CUPS-Get-Devices \-
+    localhost \- \- [01/Dec/2005:21:50:32 +0000] "GET /admin HTTP/1.1"
+      200 6667 \- \-
 
 .fi
 The \fIhost\fR field will normally only be an IP address unless you have enabled the HostNameLookups directive in the \fIcupsd.conf\fR file or if the IP address corresponds to your local machine.
 .LP
-The \fIgroup\fR field always contains "-".
+The \fIgroup\fR field always contains "\-".
 .LP
 The \fIuser\fR field is the authenticated username of the requesting user.
-If no username and password is supplied for the request then this field contains "-".
+If no username and password is supplied for the request then this field contains "\-".
 .LP
 The \fIdate-time\fR field is the date and time of the request in local time and is in the format "[DD/MON/YYYY:HH:MM:SS +ZZZZ]".
 .LP
@@ -98,7 +98,7 @@ Request too large; typically this means that a client tried to print a file larg
 Upgrading to TLS-encrypted connection.
 .TP 5
 500
-Server error; typically this happens when the server is unable to open/create a file - consult the error_log file for details.
+Server error; typically this happens when the server is unable to open/create a file \- consult the error_log file for details.
 .TP 5
 501
 The client requested encryption but encryption support is not enabled/compiled in.
@@ -110,11 +110,11 @@ HTTP version number not supported; typically this means that you have a maliciou
 The \fIbytes\fR field contains the number of bytes in the request.
 For POST requests the bytes field contains the number of bytes of non-IPP data that is received from the client.
 .LP
-The \fIipp-operation\fR field contains either "-" for non-IPP requests or the IPP operation name for POST requests containing an IPP request.
+The \fIipp-operation\fR field contains either "\-" for non-IPP requests or the IPP operation name for POST requests containing an IPP request.
 .LP
-The \fIipp-status\fR field contains either "-" for non-IPP requests or the IPP status code name for POST requests containing an IPP response.
+The \fIipp-status\fR field contains either "\-" for non-IPP requests or the IPP status code name for POST requests containing an IPP response.
 .SS ERROR LOG FILE FORMAT
-The \fIerror_log\fR file lists messages from the scheduler - errors, warnings, etc. The LogLevel directive in the
+The \fIerror_log\fR file lists messages from the scheduler \- errors, warnings, etc. The LogLevel directive in the
 .BR cupsd.conf (5)
 file controls which messages are logged:
 .nf
@@ -204,15 +204,15 @@ The format of this field is identical to the data-time field in the \fIaccess_lo
 .LP
 The \fInum-sheets\fR field provides the total number of pages (sheets) that have been printed on for the job.
 .LP
-The \fIjob-billing\fR field contains a copy of the job-billing or job-account-id attributes provided with the IPP Create-Job or Print-Job requests or "-" if neither was provided.
+The \fIjob-billing\fR field contains a copy of the job-billing or job-account-id attributes provided with the IPP Create-Job or Print-Job requests or "\-" if neither was provided.
 .LP
 The \fIjob-originating-host-name\fR field contains the hostname or IP address of the client that printed the job.
 .LP
-The \fIjob-name\fR field contains a copy of the job-name attribute provided with the IPP Create-Job or Print-Job requests or "-" if none was provided.
+The \fIjob-name\fR field contains a copy of the job-name attribute provided with the IPP Create-Job or Print-Job requests or "\-" if none was provided.
 .LP
-The \fImedia\fR field contains a copy of the media or media-col/media-size attribute provided with the IPP Create-Job or Print-Job requests or "-" if none was provided.
+The \fImedia\fR field contains a copy of the media or media-col/media-size attribute provided with the IPP Create-Job or Print-Job requests or "\-" if none was provided.
 .LP
-The \fIsides\fR field contains a copy of the sides attribute provided with the IPP Create-Job or Print-Job requests or "-" if none was provided.
+The \fIsides\fR field contains a copy of the sides attribute provided with the IPP Create-Job or Print-Job requests or "\-" if none was provided.
 .SH SEE ALSO
 .BR cupsd (8),
 .BR cupsd.conf (5),
diff --git a/man/cupsd.conf.5 b/man/cupsd.conf.5
index 00b1a22..49fdb06 100644
--- a/man/cupsd.conf.5
+++ b/man/cupsd.conf.5
@@ -446,7 +446,7 @@ The \fIAllowDH\fR option enables cipher suites using plain Diffie-Hellman key ne
 The \fIAllowRC4\fR option enables the 128-bit RC4 cipher suites, which are required for some older clients.
 The \fIAllowSSL3\fR option enables SSL v3.0, which is required for some older clients that do not support TLS v1.0.
 The \fIDenyCBC\fR option disables all CBC cipher suites.
-The \fIDenyTLS1.0\fR option disables TLS v1.0 support - this sets the minimum protocol version to TLS v1.1.
+The \fIDenyTLS1.0\fR option disables TLS v1.0 support \- this sets the minimum protocol version to TLS v1.1.
 The \fIMinTLS\fR options set the minimum TLS version to support.
 The \fIMaxTLS\fR options set the maximum TLS version to support.
 Not all operating systems support TLS 1.3 at this time.
diff --git a/man/cupsfilter.8 b/man/cupsfilter.8
index c655c21..fe4c5a2 100644
--- a/man/cupsfilter.8
+++ b/man/cupsfilter.8
@@ -126,7 +126,7 @@ command use the current user and security session. This may result in different
 The following command will generate a PDF preview of job 42 for a printer named "myprinter" and save it to a file named "preview.pdf":
 .nf
 
-    cupsfilter -m application/pdf -d myprinter -j 42 >preview.pdf
+    cupsfilter \-m application/pdf \-d myprinter \-j 42 >preview.pdf
 .fi
 .SH SEE ALSO
 .BR cups (1),
diff --git a/man/filter.7 b/man/filter.7
index dbc3150..8cb79bb 100644
--- a/man/filter.7
+++ b/man/filter.7
@@ -136,7 +136,7 @@ Sets the named keywords in the printer's PPD file. This is typically used to upd
 .TP 5
 \fBSTATE: +\fI printer-state-reason \fR[ \fI... printer-state-reason\fR ]
 .TP 5
-\fBSTATE: -\fI printer-state-reason \fR[ \fI... printer-state-reason\fR ]
+\fBSTATE: \-\fI printer-state-reason \fR[ \fI... printer-state-reason\fR ]
 Sets, adds, or removes "printer-state-reason" keywords for the current queue. Typically this is used to indicate media, ink, and toner conditions on a printer.
 .TP 5
 \fBWARNING:\fI message\fR
diff --git a/man/ippeveprinter.1 b/man/ippeveprinter.1
index fa32efb..8ab2654 100644
--- a/man/ippeveprinter.1
+++ b/man/ippeveprinter.1
@@ -143,8 +143,8 @@ If "command" is not an absolute path ("/path/to/command"),
 looks for the command in the "command" subdirectory of the CUPS binary directory, typically /usr/lib/cups/command or /usr/libexec/cups/command.
 The
 .BR cups-config (1)
-command can be used to discover the correct binary directory ("cups-config --serverbin").
-In addition, the CUPS_SERVERBIN environment variable can be used to override the default location of this directory - see the
+command can be used to discover the correct binary directory ("cups-config \--serverbin").
+In addition, the CUPS_SERVERBIN environment variable can be used to override the default location of this directory \- see the
 .BR cups (1)
 man page for more details.
 .TP 5
@@ -244,7 +244,7 @@ Logs an informational/progress message if \-v has been specified and copies the
 \fBSTATE: \fIkeyword[,keyword,...]\fR
 Sets the printer's "printer-state-reasons" attribute to the listed keywords.
 .TP 5
-\fBSTATE: -\fIkeyword[,keyword,...]\fR
+\fBSTATE: \-\fIkeyword[,keyword,...]\fR
 Removes the listed keywords from the printer's "printer-state-reasons" attribute.
 .TP 5
 \fBSTATE: +\fIkeyword[,keyword,...]\fR
diff --git a/man/ippfind.1 b/man/ippfind.1
index 32f2e6f..d77ef75 100644
--- a/man/ippfind.1
+++ b/man/ippfind.1
@@ -75,7 +75,7 @@ True is the hostname matches the given regular expression.
 .B \-l
 .TP 5
 .B \-\-ls
-Lists attributes returned by Get-Printer-Attributes for IPP printers and traditional \fIfind\fR "-ls" output for HTTP URLs.
+Lists attributes returned by Get-Printer-Attributes for IPP printers and traditional \fIfind\fR "\-ls" output for HTTP URLs.
 The result is true if the URI is accessible, false otherwise.
 .TP 5
 .B \-\-local
@@ -108,7 +108,7 @@ The result is always true.
 .B \-q
 .TP 5
 .B \-\-quiet
-Quiet mode - just returns the exit codes below.
+Quiet mode \- just returns the exit codes below.
 .TP 5
 .B \-r
 .TP 5
@@ -141,7 +141,7 @@ True if the URI matches the given regular expression.
 .TP 5
 \fB\-\-exec \fIutility \fR[ \fIargument \fR... ] \fB;\fR
 Executes the specified program if the current result is true.
-"{foo}" arguments are replaced with the corresponding value - see SUBSTITUTIONS below.
+"{foo}" arguments are replaced with the corresponding value \- see SUBSTITUTIONS below.
 .PP
 Expressions may also contain modifiers:
 .TP 5
diff --git a/man/ipptoolfile.5 b/man/ipptoolfile.5
index 7001017..5763232 100644
--- a/man/ipptoolfile.5
+++ b/man/ipptoolfile.5
@@ -166,8 +166,8 @@ The following directives are understood within a \fItest\fR:
 \fBATTR \fItag attribute-name value(s)\fR
 Adds an attribute to the test request.
 Out-of-band tags (admin-define, delete-attribute, no-value, not-settable, unknown, unsupported) have no value.
-Values for other tags are delimited by the comma (",") character - escape commas using the "\\" character.
-Common attributes and values are listed in the IANA IPP registry - see references below.
+Values for other tags are delimited by the comma (",") character \- escape commas using the "\\" character.
+Common attributes and values are listed in the IANA IPP registry \- see references below.
 .TP 5
 \fBATTR collection \fIattribute-name \fB{ MEMBER \fItag member-name value(s) ... \fB}\fR [ \fI... \fB,{ \fI... \fB} \fR]
 Adds a collection attribute to the test request.
@@ -194,7 +194,7 @@ test report.
 \fBEXPECT ?\fIattribute-name predicate(s)\fR
 .TP 5
 \fBEXPECT !\fIattribute-name\fR
-Specifies that the response must/may/must not include the named attribute. Additional requirements can be added as predicates - see the "EXPECT PREDICATES" section for more information on predicates. Attribute names can specify member attributes by separating the attribute and member names with the forward slash, for example "media\-col/media\-size/x\-dimension".
+Specifies that the response must/may/must not include the named attribute. Additional requirements can be added as predicates \- see the "EXPECT PREDICATES" section for more information on predicates. Attribute names can specify member attributes by separating the attribute and member names with the forward slash, for example "media\-col/media\-size/x\-dimension".
 .TP 5
 \fBEXPECT-ALL \fIattribute-name \fR[ \fIpredicate(s) \fR]
 .TP 5
@@ -246,7 +246,7 @@ Specifies whether
 will skip the current test if the previous test resulted in an error/failure.
 .TP 5
 \fBSTATUS \fIstatus-code \fR[ \fIpredicate\fR ]
-Specifies an expected response status-code value. Additional requirements can be added as predicates - see the "STATUS PREDICATES" section for more information on predicates.
+Specifies an expected response status-code value. Additional requirements can be added as predicates \- see the "STATUS PREDICATES" section for more information on predicates.
 .TP 5
 \fBTEST\-ID "\fIidentifier\fR"
 Specifies an identifier string for the current test.
diff --git a/man/lp.1 b/man/lp.1
index cbea3b8..839a482 100644
--- a/man/lp.1
+++ b/man/lp.1
@@ -83,7 +83,7 @@ lp \- print files
 ]
 .SH DESCRIPTION
 \fBlp\fR submits files for printing or alters a pending job.
-Use a filename of "-" to force printing from the standard input.
+Use a filename of "\-" to force printing from the standard input.
 .SS THE DEFAULT DESTINATION
 CUPS provides many ways to set the default destination. The \fBLPDEST\fR and \fBPRINTER\fR environment variables are consulted first.
 If neither are set, the current default set using the
@@ -153,7 +153,7 @@ Use a value of \fIrestart\fR with the \fI-i\fR option to restart a completed job
 \fB\-P \fIpage-list\fR
 Specifies which pages to print in the document.
 The list can contain a list of numbers and ranges (#-#) separated by commas, e.g., "1,3-5,16".
-The page numbers refer to the output pages and not the document's original pages - options like "number-up" can affect the numbering of the pages.
+The page numbers refer to the output pages and not the document's original pages \- options like "number-up" can affect the numbering of the pages.
 .SS COMMON JOB OPTIONS
 Aside from the printer-specific options reported by the
 .BR lpoptions (1)
@@ -183,7 +183,7 @@ Prints the job in reverse portrait (rotated 180 degrees).
 \fB\-o print\-quality=4\fR
 .TP 5
 \fB\-o print\-quality=5\fR
-Specifies the output quality - draft (3), normal (4), or best (5).
+Specifies the output quality \- draft (3), normal (4), or best (5).
 .TP 5
 \fB\-o sides=one\-sided\fR
 Prints on one side of the paper.
@@ -202,19 +202,19 @@ The \fI-q\fR option accepts a different range of values than the Solaris lp comm
 Print two copies of a document to the default printer:
 .nf
 
-    lp -n 2 filename
+    lp \-n 2 filename
 
 .fi
 Print a double-sided legal document to a printer called "foo":
 .nf
 
-    lp -d foo -o media=legal -o sides=two-sided-long-edge filename
+    lp \-d foo \-o media=legal \-o sides=two-sided-long-edge filename
 
 .fi
 Print a presentation document 2-up to a printer called "bar":
 .nf
 
-    lp -d bar -o number-up=2 filename
+    lp \-d bar \-o number-up=2 filename
 .fi
 .SH SEE ALSO
 .BR cancel (1),
diff --git a/man/lpadmin.8 b/man/lpadmin.8
index 1b8c91b..ce6a698 100644
--- a/man/lpadmin.8
+++ b/man/lpadmin.8
@@ -120,7 +120,7 @@ command.
 .TP 5
 \fB\-o \fIname\fB-default=\fIvalue\fR
 Sets a default server-side option for the destination.
-Any print-time option can be defaulted, e.g., "-o number-up-default=2" to set the default "number-up" option value to 2.
+Any print-time option can be defaulted, e.g., "\-o number-up-default=2" to set the default "number-up" option value to 2.
 .TP 5
 \fB\-o port\-monitor=\fIname\fR
 Sets the binary communications program to use when printing, "none", "bcp", or "tbcp".
@@ -162,7 +162,7 @@ If the resulting class becomes empty it is removed.
 Sets user-level access control on a destination.
 Names starting with "@" are interpreted as UNIX groups.
 The latter two forms turn user-level access control off.
-Note: The user 'root' is not granted special access - using "-u allow:foo,bar" will allow users 'foo' and 'bar' to access the printer but NOT 'root'.
+Note: The user 'root' is not granted special access \- using "\-u allow:foo,bar" will allow users 'foo' and 'bar' to access the printer but NOT 'root'.
 .TP 5
 \fB\-v "\fIdevice-uri\fB"\fR
 Sets the \fIdevice-uri\fR attribute of the printer queue.
@@ -218,7 +218,7 @@ These files should not be edited directly and are an implementation detail of CU
 Create an IPP Everywhere print queue:
 .nf
 
-    lpadmin -p myprinter -E -v ipp://myprinter.local/ipp/print -m everywhere
+    lpadmin \-p myprinter \-E \-v ipp://myprinter.local/ipp/print \-m everywhere
 
 .fi
 .SH SEE ALSO
diff --git a/man/lpoptions.1 b/man/lpoptions.1
index a34fd01..7550d9a 100644
--- a/man/lpoptions.1
+++ b/man/lpoptions.1
@@ -102,9 +102,9 @@ Destinations can only be removed using the
 .BR lpadmin (8)
 command.
 .SH FILES
-\fI~/.cups/lpoptions\fR - user defaults and instances created by non-root users.
+\fI~/.cups/lpoptions\fR \- user defaults and instances created by non-root users.
 .br
-\fI/etc/cups/lpoptions\fR - system-wide defaults and instances created by the root user.
+\fI/etc/cups/lpoptions\fR \- system-wide defaults and instances created by the root user.
 .SH CONFORMING TO
 The \fBlpoptions\fR command is unique to CUPS.
 .SH SEE ALSO
diff --git a/man/lpr.1 b/man/lpr.1
index f367cd9..635d126 100644
--- a/man/lpr.1
+++ b/man/lpr.1
@@ -138,7 +138,7 @@ Prints the job in reverse portrait (rotated 180 degrees).
 \fB\-o print\-quality=4\fR
 .TP 5
 \fB\-o print\-quality=5\fR
-Specifies the output quality - draft (3), normal (4), or best (5).
+Specifies the output quality \- draft (3), normal (4), or best (5).
 .TP 5
 \fB\-o sides=one\-sided\fR
 Prints on one side of the paper.
@@ -154,19 +154,19 @@ The \fI\-c\fR, \fI\-d\fR, \fI\-f\fR, \fI\-g\fR, \fI\-i\fR, \fI\-n\fR, \fI\-t\fR,
 Print two copies of a document to the default printer:
 .nf
 
-    lpr -# 2 filename
+    lpr \-# 2 filename
 
 .fi
 Print a double-sided legal document to a printer called "foo":
 .nf
 
-    lpr -P foo -o media=legal -o sides=two-sided-long-edge filename
+    lpr \-P foo \-o media=legal \-o sides=two-sided-long-edge filename
 
 .fi
 Print a presentation document 2-up to a printer called "foo":
 .nf
 
-    lpr -P foo -o number-up=2 filename
+    lpr \-P foo \-o number-up=2 filename
 .fi
 .SH SEE ALSO
 .BR cancel (1),
diff --git a/man/mime.convs.5 b/man/mime.convs.5
index 25a6f76..04ba394 100644
--- a/man/mime.convs.5
+++ b/man/mime.convs.5
@@ -35,11 +35,11 @@ A value of 100 means that the filter uses a large amount of resources while a va
 The \fIfilter\fR field specifies the filter program filename.
 Filenames are relative to the CUPS filter directory.
 .SH FILES
-\fI/etc/cups\fR - Typical CUPS configuration directory.
+\fI/etc/cups\fR \- Typical CUPS configuration directory.
 .br
-\fI/usr/lib/cups/filter\fR - Typical CUPS filter directory.
+\fI/usr/lib/cups/filter\fR \- Typical CUPS filter directory.
 .br
-\fI/usr/libexec/cups/filter\fR - CUPS filter directory on macOS.
+\fI/usr/libexec/cups/filter\fR \- CUPS filter directory on macOS.
 .SH EXAMPLES
 Define a filter that converts PostScript documents to CUPS Raster format:
 .nf
diff --git a/man/mime.types.5 b/man/mime.types.5
index 7e83de1..3572726 100644
--- a/man/mime.types.5
+++ b/man/mime.types.5
@@ -35,7 +35,7 @@ See the "TYPE MATCHING AND PRIORITY" section for more information.
 .LP
 The rules may be grouped using parenthesis, joined using "+" for a logical AND, joined using "," or whitespace for a logical OR, and negated using "!".
 .SS RULES
-Rules take two forms - a filename extension by itself and functions with test
+Rules take two forms \- a filename extension by itself and functions with test
 values inside parenthesis.
 The following functions are available:
 .TP 5
@@ -85,7 +85,7 @@ extension "doc", normally the type "text/bar" will be chosen since its name is
 alphanumerically smaller than "text/foo".
 However, if "text/foo" also defines a higher priority than "text/bar", "text/foo" will be chosen instead.
 .SH FILES
-\fI/etc/cups\fR - Typical CUPS configuration directory.
+\fI/etc/cups\fR \- Typical CUPS configuration directory.
 .SH EXAMPLES
 Define two MIME media types for raster data, with one being a subset with higher priority:
 .nf
diff --git a/man/ppdc.1 b/man/ppdc.1
index 44e9d24..38e2669 100644
--- a/man/ppdc.1
+++ b/man/ppdc.1
@@ -85,7 +85,7 @@ The default is to generate uncompressed PPD files.
 \fB\-\-crlf\fR
 .TP 5
 \fB\-\-lf\fR
-Specifies the line ending to use - carriage return, carriage return and line feed, or line feed alone.
+Specifies the line ending to use \- carriage return, carriage return and line feed, or line feed alone.
 The default is to use the line feed character alone.
 .SH NOTES
 PPD files are deprecated and will no longer be supported in a future feature release of CUPS.
